### PR TITLE
aranea/nethack

### PR DESCRIPTION
**Pull request by atlaua**
Some updates for games-roguelike/nethack.

I'm the maintainer.

### CODE REVIEW

**Comment by atlaua** [comment]
I guess this syntax didn't exist back in 2006 when this ebuild was written. Anyway, I'll fix it.

**Comment by atlaua** [comment]
I pushed a fix (and rebased the patchset while I was at it).

**Comment by hasufell** [comment]
> I guess this syntax didn't exist back in 2006 when this ebuild was written.

it works even with bash-2.05b

File: games-roguelike/nethack/nethack-3.4.3-r5.ebuild [hunk 1]
```diff
@@ -30,7 +30,7 @@ DEPEND="${RDEPEND}
 	)"
 
 BINDIR="/usr/games/bin"
-HACKDIR="/usr/share/games/${PN}"
+HACKDIR="/usr/lib/${PN}"
```
Comment (hasufell) [inline games-roguelike/nethack/nethack-3.4.3-r5.ebuild]: Mh no... we should not install into this dir and it also causes my PM to throw warnings.
We might even remove the `lib` directory completely sometime in the future, so this has to be `HACKDIR="/usr/$(get_libdir)/${PN}"`, but I'd avoid `$(get_libdir)` in global scope, although HACKDIR is not part of metadata. Moving it to pkg_setup is probably cleaner and will still be available in all functions.

Comment (atlaua) [inline games-roguelike/nethack/nethack-3.4.3-r5.ebuild]: I've addressed your concerns and pushed an updated patchset.

**Comment by monsieurp** [comment]
I'm gonna merge this PR.

**Comment by monsieurp** [comment]
Thank you.

**Comment by idella** [comment]
Julian is this good to go?  Oh it went

**Pull request merged**